### DISCUSSION TITLE
Eddystone EID

### DISCUSSION THREAD

**Discussion by gnome4711** [ESPresense/ESPresense#599] [General]
Hi there!

In the 3xx beta's  is possible the enter a key to resolve BLE identities :'Known BLE identity resolving keys, should be 32 hex chars space seperated:'
Is this suposed to work with eddystones EID beacons? I gave it a try and setup a eddystone eid beacon on android. I entered the key in the espresense, but the beacon does not show up. Instead a beacon shoes up with the id 'ad:0xfeaa'. This id doesn't change by changing the identity key.
This would be a great feature.

**Comment by DTTerastar** [comment]
[Eddystone-EID](https://github.com/google/eddystone/blob/master/eddystone-eid)
The Eddystone-EID frame broadcasts an encrypted ephemeral identifier that changes periodically at a rate determined during the initial registration with a web service. The broadcast ephemeral ID can be resolved remotely by the service with which it was registered, but to other observers appears to be changing randomly. This frame type is intended for use in security and privacy-enhanced devices.

It doesn't sound like that's an IRK.  I would need more info to be able and support that.  What webserver are you registering it with?
